Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -38,6 +38,9 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
         this._basicPropertyNames = ["font-size", "font-style", "font-weight", "font-stretch"];
 
         this._abortController = new AbortController;
+        this._throttledUpdate = new Throttler(() => { this.update() }, 250);
```

Comment:
I also think this is too slow, since it will also affect programmatic changes an author might be making. Only updating every quarter of a second will feel like a performance issue. I'd again consider 50-100ms max here.

Reply:
You can argue that the Fonts panel should react more frequently to style changes caused outside of it to give the illusion of performance, while, ironically, degrading it a bit :) 

I'm skeptical that reflecting instant updates is the intent of someone monitoring the Fonts panel while the page styles are actively changing, user-driven or scripted.

When making high-frequency changes in the Styles panel (increment/decrement using the arrow keys), my hypothesis is that a developer is monitoring the page itself to observe the effects and the value in the Styles panel to know when to stop. A slight delay to another panel, the Fonts panel, is likely not their focus. Even so, throttling ensures the panel updates at reasonably quick intervals.

If you prefer, we can lower the throttling rate to `100ms`.

I'm not too vested in this throttling as much as I am with preventing reacting to style changes that originate from the Fonts panel, the `this._skipNextUpdate` flag. Range inputs are a source of high-frequency changes we want to control.